package.json

```diff
@@ -4,10 +4,10 @@
    "description": "Projeto de backend no onBoard da TaqTile",
    "main": "index.js",
    "scripts": {
-      "test": "echo \"Error: no test specified\" && exit 1",
       "start": "ts-node src/index.ts",
+      "build": "npx tsc",
       "lint": "npx eslint src/**",
-      "typeorm": "typeorm-ts-node-commonjs"
+      "test": "npx mocha test/**/*.spec.ts"
```

Comment:
O `npx` é mais indicado pra usos únicos, ou seja, quando vc n tem os pacotes já instalados. Aqui a gente tem eles instalados por conta de como o package.json funciona, então vc poderia fazer todos esses comandos sem o `npx`. Mais detalhes [aqui](https://www.geeksforgeeks.org/what-are-the-differences-between-npm-and-npx/#:~:text=NPX%3A%20The%20npx%20stands%20for,without%20even%20installing%20that%20package.), por exemplo.

Outra coisa: provavelmente o npx vai instalar as versões mais recentes das bibliotecas, o que pode cusar problemas de compatibilidade conforme eles vão atualizando.

Reply:
Feito alterações, npx retirado.